src/layers/legacy/medImageIO/itkDCMTKImageIO.cpp

```diff
@@ -679,15 +678,6 @@ double DCMTKImageIO::GetPositionOnStackingAxisForImage (int index)
     closestAxis[0] = round(m_Direction[2][0]);
     closestAxis[1] = round(m_Direction[2][1]);
     closestAxis[2] = round(m_Direction[2][2]);
```

Comment:
Maybe best to determine which direction is the closest to an axis by determining the max of m_Direction[2] instead of removing juste the if.